update-xars.sh

```diff
@@ -0,0 +1,49 @@
+VERSION=$1
+DIR=$2
```

Comment:
I think this is unnecessary. Just create a tmp folder with `mktemp -d` and don't bother pushing this around as a parameter.

Reply:
good thing when pushing this around is that the script can be used standalone when preparing a instance with some additional custom XARs. This way you could run it before starting the db, add your custom XARs to the same folder and `-v YOUR-FOLDER:/opt/exist/autodeploy` when calling `docker run`